brage-import/src/test/java/no/sikt/nva/brage/migration/testutils/ReferenceGenerator.java

```diff
@@ -293,7 +294,7 @@ private static Reference buildReference(Builder builder) {
             }
             if (NvaType.VISUAL_ARTS.getValue().equals(builder.getType().getNva())) {
                 return new Reference.Builder().withPublicationInstance(
-                        new VisualArts(VisualArtsSubtype.createOther(null), null, null))
```

Comment:
round tripping via Jackson gives you null value, but not you get empty sets or list